samza-api/src/main/java/org/apache/samza/config/Config.java

```diff
@@ -154,6 +155,9 @@ public double getDouble(String k) {
       return defaultValue;
 
     String value = get(k);
+    if (value.isEmpty()) {
+      return Collections.emptyList();
```

Comment:
This case is no different than the case where the key is absent. Unless we want to throw some exception or validation error, I recommend returning `defaultValue`